### DISCUSSION TITLE
Langchain not found

### DISCUSSION THREAD

**Discussion by tpmeehan** [imartinez/privateGPT#317] [Q&A]
Somehow after doing the installation, when executing 'python ingest.py' I get the following error response:

File "/Users/tommeehan/privateGPT/ingest.py", line 6, in <module>
    from langchain.document_loaders import (
ModuleNotFoundError: No module named 'langchain'

It seems like langchain was installed.  This line was generated during the installation process

_Requirement already satisfied: langchain==0.0.171 in /usr/local/lib/python3.9/site-packages (from -r requirements.txt (line 1)) (0.0.171)_

this is what I get when running langchain environment:

LangChain Environment:
library_version:0.0.171
platform:macOS-10.16-x86_64-i386-64bit
runtime:python
runtime_version:3.9.2

Anyone have any idea how to troubleshoot this issue.  Thanks

**Comment by imartinez** [comment]
Apparently the dependency was not properly installed. I'd suggest you to create a python venv for the project, activate it and then install the dependencies there. That will ensure a clean installation and avoid incompatibilities. Remember to always activate the environment whenever you go back to the folder to execute privateGPT

**Reply by tpmeehan** [reply]
Thank you.  I will give that a try
